addons/hr_expense/models/account_move.py

```diff
@@ -46,7 +46,7 @@ def _creation_message(self):
     def _compute_payment_state(self):
         company_paid = self.filtered(lambda m: m.expense_sheet_id.payment_mode == 'company_account')
         for move in company_paid:
-            move.payment_state = move._get_invoice_in_payment_state()
+            move.payment_state = 'paid'
```

Comment:
> You need to keep the in_payment state until the payment is matched with a bank transaction.

In case of company paid expenses we are setting the `payment_state` as `paid` at creation.
https://github.com/odoo/odoo/blob/33198b3443829d6d6e421cf9d6b8033a040b5263/addons/hr_expense/models/hr_expense.py#L1060
What's the point of reverting it back to `in_payment`?

---

> Btw, the test is passing without this diff

Just in community, because the `in_payment` state is defined in `account_accountant`
https://github.com/odoo/enterprise/blob/1a43c8cb767b49ce7697815fc6e4573c4e216402/account_accountant/models/account_move.py#L18
With `account_accountant` installed it fails successfully without the diff